include/deal.II/fe/fe.h

```diff
@@ -1574,6 +1574,23 @@ class FiniteElement : public Subscriptor,
   element_multiplicity (const unsigned int index) const;
 
   /**
+    * Return a reference to a child finite element that matches the components
+    * selected by the given ComponentMask @p mask.
+    *
+    * For an arbitrarily nested FESystem, this function returns the inner-most
+    * FiniteElement that matches the given mask. The method fails if the @p mask
+    * does not exactly match one of the contained finite elements. It is most
+    * useful if the current object is an FESystem, as the return value can
+    * only be @p this in all other cases.
+    *
+    * Note that the returned object can be an FESystem if the
+    * mask matches it but not any of the contained objects.
+    */
```

Comment:
Can you add a note what happens if the operation is not possible? Or have you decided on a design where this is *always* possible?